realm/realm-library/src/testUtils/java/io/realm/entities/AllTypes.java

```diff
@@ -108,36 +108,36 @@
     private RealmList<UUID> columnUUIDList;
     private RealmList<Mixed> columnMixedList;
 
-    private RealmSet<Boolean> columnBooleanSet;
-    private RealmSet<String> columnStringSet;
-    private RealmSet<Integer> columnIntegerSet;
-    private RealmSet<Float> columnFloatSet;
-    private RealmSet<Long> columnLongSet;
-    private RealmSet<Short> columnShortSet;
-    private RealmSet<Double> columnDoubleSet;
-    private RealmSet<Byte> columnByteSet;
-    private RealmSet<byte[]> columnBinarySet;
-    private RealmSet<Date> columnDateSet;
-    private RealmSet<ObjectId> columnObjectIdSet;
-    private RealmSet<UUID> columnUUIDSet;
-    private RealmSet<Decimal128> columnDecimal128Set;
-    private RealmSet<DogPrimaryKey> columnRealmModelSet;
-    private RealmSet<Owner> columnRealmModelNoPkSet;
-    private RealmSet<Mixed> columnMixedSet;
-
-    @Required private RealmSet<Boolean> columnRequiredBooleanSet;
-    @Required private RealmSet<String> columnRequiredStringSet;
-    @Required private RealmSet<Integer> columnRequiredIntegerSet;
-    @Required private RealmSet<Float> columnRequiredFloatSet;
-    @Required private RealmSet<Long> columnRequiredLongSet;
-    @Required private RealmSet<Short> columnRequiredShortSet;
-    @Required private RealmSet<Double> columnRequiredDoubleSet;
-    @Required private RealmSet<Byte> columnRequiredByteSet;
-    @Required private RealmSet<byte[]> columnRequiredBinarySet;
-    @Required private RealmSet<Date> columnRequiredDateSet;
-    @Required private RealmSet<ObjectId> columnRequiredObjectIdSet;
-    @Required private RealmSet<UUID> columnRequiredUUIDSet;
-    @Required private RealmSet<Decimal128> columnRequiredDecimal128Set;
+//    private RealmSet<Boolean> columnBooleanSet;
```

Comment:
Should be remove? Or maybe I don't get the PR comment, but if there is any reason for keeping these, please also add a comment here.

Reply:
These are leftovers, forgot to remove them.